kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java

```diff
@@ -62,6 +62,7 @@
 
     private static final String FUNCTION_TYPE = "function";
     private static final String FUNCTION_ARGS = "args";
+    private static final String FUNCTION_LABLEVAlUE = "lableValue";
```

Comment:
perhaps FUNCTION_LABEL_VALUE?

Reply:
@cristianonicolai Updated, Please review agian.

Reply:
@bxf12315, could you look into adding some tests, please